config/filter.d/apache-auth.conf

```diff
@@ -24,6 +24,8 @@ failregex = ^client (?:denied by server configuration|used wrong authentication
             ^%(auth_type)sunknown algorithm `(?:[^']*|.*?)' received\b
             ^invalid qop `(?:[^']*|.*?)' received\b
             ^%(auth_type)sinvalid nonce .*? received - user attempted time travel\b
+            ^Hostname .* provided via SNI(, but no hostname| and hostname .*) provided\b
```

Comment:
Please replace the catch-alls `.*` with `\S+`, and use non-capturing groups `(?: ... )`here:
```diff
- ^Hostname .* provided via SNI(, but no hostname| and hostname .*) provided\b
+ ^Hostname \S+ provided via SNI(?:, but no hostname| and hostname \S+) provided\b
```